client/history/client.go

```diff
@@ -1048,7 +1049,7 @@ func (c *clientImpl) getClientForDomainID(domainID string) (Client, error) {
 }
 
 func (c *clientImpl) getClientForShardID(shardID int) (Client, error) {
-	client, err := c.clients.GetClientForKey(string(shardID))
+	client, err := c.clients.GetClientForKey(strconv.Itoa(shardID))
```

Comment:
this doesn't go into any load-distribution system, just an in-memory client cache.  safe to change.